### ISSUE TITLE
Helpful Rework/Ehancement

### ISSUE THREAD

**Issue by thesehands** [RaspberryPints/RaspberryPints#4]
So I have a rework of all the CSS as well as a test database to upload is there a place that wont affects anyone's work? That way people can preview it if they want and let me know what they think.

My thoughts are that the initial setup is over engineered to a certain extent and its important to reduce the code as much as possible while remaining fairly flexible. I made some tweaks so that it would auto size to the monitor. I'm still working a few things, but I made some progress. On a side note since I do not setup hosting solutions and am not a great networking person. Would it be possible to make the site available to any one that is connect to your wifi with the hops of eventually creating a mobile friendly version of this?

**Comment by thadius856** [comment]
The easy way to do this would be to Fork the repository (see here for how: https://help.github.com/articles/fork-a-repo).

Make your changes to that fork, then initiate a Pull Request (https://help.github.com/articles/creating-a-pull-request).

I'll test and review, comment as necessary, then merge back into the master once everything's ready to go.

**Comment by thesehands** [comment]
I forked the repo and added my files

**Comment by thadius856** [comment]
Ok man, I just reviewed what you did.

Having some problems on this end. I imported kegerface.sql using the phpMyAdmin web interface. It created database 'kegerface' with table 'beers'. This matches what I'm reading in the php section before the opening html tag.

However, I'm getting no output from the database on the page load. No idea why. I can see 3 beers in the 'beers' table.

1000 words:

![untitled](https://f.cloud.github.com/assets/6320180/1865356/0fc33a60-780d-11e3-82d6-9268cecc11bf.png)
